policy/modules/system/lvm.fc

```diff
@@ -21,6 +21,9 @@
 # /usr
 #
 /usr/bin/cryptsetup		--	gen_context(system_u:object_r:lvm_exec_t,s0)
+ifdef(`distro_gentoo',`
+/usr/bin/dmeventd		--	gen_context(system_u:object_r:lvm_exec_t,s0)
+')
```

Comment:
Please move to line 76 (after all the `/usr/bin` entires), with empty lines before and after.